ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -586,7 +572,6 @@ export default class ConfirmTransactionBase extends Component {
             renderSimulationFailureWarning ? null : () => this.handleEditGas()
```

Comment:
If we move the `renderSimulationFailureWarning` to encapsulate the `TransactionDetail` component, then we can also remove this ternary operator.

Reply:
https://github.com/MetaMask/metamask-extension/pull/23886/commits/cb632f17edb983b176767be12805d9137dc7beea